src/VirtoCommerce.SubscriptionModule.Data/VirtoCommerce.SubscriptionModule.Data.csproj

```diff
@@ -13,6 +13,7 @@
     <ProjectReference Include="..\VirtoCommerce.SubscriptionModule.Core\VirtoCommerce.SubscriptionModule.Core.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="FluentValidation" Version="8.6.2" />
```

Comment:
please check that this version doesn't higher than used in the platform or other modules, to avoid the side effect of an unexpected upgrade in the runtime.

Reply:
We are using 8.6.3 in the platform, so it is ok.